integrations/anthropic/tests/test_chat_generator.py

```diff
@@ -703,7 +726,7 @@ def test_convert_streaming_chunks_to_chat_message_with_multiple_tool_calls(self)
                 meta={
                     "type": "message_delta",
                     "delta": {"stop_reason": "tool_calls", "stop_sequence": None},
-                    "usage": {"completion_tokens": 40},
+                    "usage": {"completion_tokens": 40, "output_tokens": 40},
```

Comment:
I'm not sure `output_tokens` should be added here. I believe Anthropics name for `output_tokens` is `completion_tokens`